### ISSUE TITLE
Permissions Override?

### ISSUE THREAD

**Issue by JericM** [maker-dude/EpicZones#10]
My issue- I would like to disable admin/owner overrides in Epiczones. 
Steps-
1- I create a zone for a user (I'm the owner)
2- I set up their flags and make them owner.
3- I set default build/destroy deny
4- I set USER build/destroy allow
5- I set owner as USER.

My issue is: I, as the only owner, do not want to be able to destroy or build in any zone which i do not own. Is there a way to do this?

Plugins I use:
1- Essentials/Chat/Groupmanager/GroupBridge/Protect/Spawn
2- Lockette
3- Epiczones
4- Vault

**Comment by CourtArtanis** [comment]
Yes, just negate the epiczones.ignorepermissions node

-epiczones.ignorepermissions 

(under your group)

**Comment by JericM** [comment]
That did not work :(

King:
    default: false
    permissions:
    - -epiczones.ignorepermissions
    inheritance:
    - royalty
    - g:groupmanager_admin
    - g:bukkit_admin
    - g:essentials_admin
    info:
      prefix: '&2[King]'
      build: true
      suffix: ''

**Comment by JericM** [comment]
It was working a while back, and then I updated some permissions files and it all stopped working. The zones work, just not the Build/Destroy permissions. It's like it's ignoring them.

**Comment by JericM** [comment]
Any ideas?

**Comment by CourtArtanis** [comment]
Alright, make sure your groups don't have the operator node.

" - '*' " etc.

If you do this, you will default to Op everytime and ignore 99% of any permission, negated or not.

Ex. 

King:
default: false
permissions:
- -epiczones.ignorepermissions
- epiczones.*

This setup should work.

Ex 2.

King:
default: false
permissions:
- -epiczones.ignorepermissions
- '*'

This setup will not work in 99% of cases. -'*' node is the Operator node. It's practically the same as adding yourself to ops.txt - Operators > all plugins and nodes

Let me know if this is the issue.

**Comment by JericM** [comment]
Well, I deleted all plugins and refreshed everything, and redrew all zones. It works now. This is my setup, in case anyone else needs it:

The only permissions node for EZones is the one I added to Essentials Admin list:
g:essentials_admin:
    permissions:
    - -essentials.backup
    - -essentials.essentials
    - -essentials.setspawn
    - -essentials.reloadall
    - -essentials.plugin
    - essentials.*
    - -epiczones.ignorepermissions

My group names, Serf, Citizen, Elite and Sovereign, work better when setting up permission. 
So when I create a zone, I set permissions Serf build/destroy deny (which everyone inherits).
Then I simply give specific usernames permission. 

Now, another slightly related question that doesn't merit a ticket:

Can I remove permissions in game? I noticed when I typed  /zone perm remove (after editing the zone) it says "Permissions Removed" but didn't actually do anything. Is this intended?
